src/components/asset-list/RecyclerAssetList2/ClaimablesListHeader.tsx

```diff
@@ -90,4 +90,4 @@ ClaimablesListHeader.animationDuration = TokenFamilyHeaderAnimationDuration;
 
 ClaimablesListHeader.height = TokenFamilyHeaderHeight;
 
-export default ClaimablesListHeader;
+export default React.memo(ClaimablesListHeader);
```

Comment:
I'd rather this be a named export too, but at least declaring `ClaimablesListHeader` as a named function `function ClaimablesListHeader` instead of `const ClaimablesListHeader =` makes it so we have better stack traces in case of issues

Reply:
i didn't know that about named functions, thanks!